### PR TITLE
add FFT post filter info

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Review by andyp1per — CHANGES_REQUESTED** [review]

File: common/source/docs/common-imu-fft.rst [hunk 1]
```diff
@@ -26,6 +26,29 @@ For most uses with other FFT related advanced parameters at their default, this
...
+
+Motor Noise Check
+-----------------
+
+If bit 1 of :ref:`FFT_OPTIONS<FFT_OPTIONS>` is set, then the measurement window for the FFT is centered about the motor frequency as reported by ESC telemetry. This allows an easy way to check the noise using the FTN2 log messages. See :ref:`logmessages` FTN2 section. This bit must be used with bit 0 also set.
+
+Optimal Use
+===========
+
+The best use of the FFT notch filter is in addition to other dynamic harmonic notch filter (Throttle, ESC, or RPM based), using the post filter chain :ref:`FFT_OPTIONS<FFT_OPTIONS>` bit.
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-imu-fft.rst:47]: why not the other two??

File: common/source/docs/common-imu-fft.rst [hunk 2]
```diff
@@ -26,6 +26,29 @@ For most uses with other FFT related advanced parameters at their default, this
...
 
+FFT Options
+===========
+
+There are two options which can be selected by setting the appropriate bit in the :ref:`FFT_OPTIONS<FFT_OPTIONS>` parameter that affect FFT operation:
+
+Post Filter Chain FFT Analysis Window
+-------------------------------------
+
+Normally, the FFT analysis for adjusting the center frequency is done by measuring the noise directly at the input to the FFT notch filter. However, if bit 0 of :ref:`FFT_OPTIONS<FFT_OPTIONS>` is set, then the measurement window is taking into account the effects of the low pass filter that is after the notch filter(s). This is useful if there is high frequency noise, which impacts the control response less than lower frequency noise due to the low pass at the end of the filter chain, but may be targeted by the FFT measurement. Setting this bit will only track those lower frequency, and more critical, noise peaks.
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-imu-fft.rst:37]: done

File: common/source/docs/common-imu-fft.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (andyp1per) [reply common/source/docs/common-imu-fft.rst:47]: don't understand your question - other two what?

[the remark this answers is not in the document]
Comment (andyp1per) [reply common/source/docs/common-imu-fft.rst:47]: The FFT filter can be used without post-filter - your text implies that you should always use it post filter. Maybe something like: "The best post-filter use of the FFT notch filter.."

[the remark this answers is not in the document]
Comment (andyp1per) [reply common/source/docs/common-imu-fft.rst]: Still not agreeing with the text implication that the best use for FFT notch is post-filter. FFT notch is a tool that can be used effectively when the circumstances allow/dictate - just like ESC or Throttle based notch

[the remark this answers is not in the document]
Comment (andyp1per) [reply common/source/docs/common-imu-fft.rst]: "The using the FFT notch filter post-filter works best in conjunction with another notch filter driven by an alternative mechanism such as throttle, ESC, or RPM using the post filter chain :ref:`FFT_OPTIONS<FFT_OPTIONS>` bit."

[the remark this answers is not in the document]
Comment (andyp1per) [reply common/source/docs/common-imu-fft.rst]: I don't think its possible to say what the optimum configuration is - it really depends on the vehicle.

In terms of the wording I think the word "filter" is messing with you. We could use the term "post-LPF" to determine position and then "notch filter" might make more sense

[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-imu-fft.rst]: okay another try at it

**Pull request merged**